### ISSUE TITLE
using SDA on smartphone?

### ISSUE THREAD

**Issue by indio68** [Jessecar96/SteamDesktopAuthenticator#712]
Greetings, i would like to know if i can add SDA and use it on my smartphone now that i have one. thanks if yes how? i didnìt see any android version

**Comment by Wolterhon** [comment]
Nope, but you can use Steam's official app, I think it supports multiple accounts.

**Comment by Famous5000** [comment]
> Greetings, i would like to know if i can add SDA and use it on my smartphone now that i have one. thanks if yes how? i didnìt see any android version

If you have a smart phone, it's very strongly recommended that you deauthenticate this app, then set up your phone using the official Steam App. This is more secure and will help to ensure that your account is safe.
